integr8ly/keycloak-operator
# waiting for postgres

Provision postgresql first, wait for it to come up, then deploy the application with the right credentials for the data layer.

## Verification
- Create namespace on openshift 4
- Deploy operator
- Create a keycloak CR with `provision: true`
- see that keycloak comes up healthy and functional.
